crates/pow/README.md

```diff
@@ -38,12 +38,25 @@ A challenge solution is valid when:
 
 ## Dynamic Difficulty
 
-The system automatically adjusts challenge difficulty based on usage:
-- **Target calculation**: `max_target / difficulty`
-- **Max target**: `u64::MAX >> baseline`
-- **Difficulty**: `max(num_active_challenges << growth_rate, 1)`
+The system automatically adjusts challenges difficulty based on usage and the request complexity. 
 
-This means as more users solve challenges, the difficulty increases exponentially, providing automatic rate limiting.
+The challenge target $t$ is computed as follows: 
+$$
+t = \dfrac{2^{64}} {d}
+$$
```

Comment:
This still doesn't render correctly - I believe there should be a newline before `$$`.